src/nvim/generators/gen_options.lua

```diff
@@ -191,7 +205,7 @@ w('  [' .. ('%u'):format(#options.options) .. ']={.fullname=NULL}')
 w('};')
 w('')
 
-for k, v in pairs(defines) do
+for k, v in pairsByKeys(defines) do
```

Comment:
Instead of introducing an artificial sort operation, we can solve the _root cause_ by instead doing `table.insert(defines, { 'PV_' .. varname:sub(3):upper(),  pv_name}` and then use `ipairs` completely normally here.